### ISSUE TITLE
Add GA tags

### ISSUE THREAD

**Issue by gammaw** [molnar-a/lahmacun-app#2]
See 

- https://github.com/mmmnmnm/lahmacun/blob/0a9a61b033c0af3f23c2510f39ba5cdc66dfad44/wp/wp-content/themes/lahma_maker/header.php#L25-L31 (ID) and 
- https://github.com/mmmnmnm/lahmacun/blob/0a9a61b033c0af3f23c2510f39ba5cdc66dfad44/wp/wp-content/themes/lahma_maker/assets/playerfiles/radio_player.vue#L345-L366 (radio play) and 
- https://github.com/mmmnmnm/lahmacun/blob/0a9a61b033c0af3f23c2510f39ba5cdc66dfad44/wp/wp-content/themes/lahma_maker/js/scripts.js#L83-L104 (arcsi). 

Note: we wouldn't want to got with GA4 tags (which we tried for ananasz) but rather stick with UA for the time being.

**Comment by molnar-a** [comment]
version: 1.0.6

**Comment by molnar-a** [comment]
@gammaw about the radio play snippet: is that how it supposed to work? if a user starts the radio when a jingle is playing, it won't update later? or am i missing something?

**Comment by gammaw** [comment]
> if a user starts the radio when a jingle is playing, it won't update later? or am i missing something?

@molnar-a you're right, currently we'll record it as +1 one for the corresponding jingle playlist (if the listener stays for hours over multiple shows, we won't record it for the respective shows). True, that's a limitation which we should improve in the future. 

FYI @pvj @szkjn
